_inc/client/writing/theme-enhancements.jsx

```diff
@@ -13,8 +12,8 @@ import { FormFieldset } from 'components/forms';
 import { ModuleToggle } from 'components/module-toggle';
 import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
 import { ModuleSettingCheckbox } from 'components/module-settings/form-components';
-import SettingsCard from 'components/settings-card';
-import SettingsGroup from 'components/settings-group';
+import { SettingsCard } from 'components/settings-card';
+import { SettingsGroup } from 'components/settings-group';
```

Comment:
@beaulebens this change was causing a major JS error on Writing tab. I've reverted it to the way I wrote it, without the curly braces since the imported component is the default exported.